hack/operatorhub/publish-bundle.sh

```diff
@@ -77,7 +77,7 @@ cp -a "${BUNDLE_DIR}"/. "${OPERATORS_DIR}"
 
 TITLE="operators ${OPERATOR_NAME} (${OPERATOR_VERSION})"
 skipInDryRun git add .
-skipInDryRun git commit -m"${TITLE}"
+skipInDryRun git commit -s -m"${TITLE}"
```

Comment:
currently maistra-bot email is null I think. I hope this won't be an issue.

for instance, we see in the automated pr's: `Co-authored-by: maistra-bot <null>`